ppmTim.properties

```diff
@@ -12,7 +12,7 @@ privacy.redaction.id.included=BEA10000,BEA10001
 
 # Configuration details for geofencing.
 privacy.filter.geofence=OFF
```

Comment:
We removed the TIM message from PPM in another PR right? should we grab those changes and remove this properties file altogether?

Reply:
That's correct, good call. I've merged dev into the feature branch and removed the `ppmTIm.properties` file.